### PR TITLE
Improve SAML configuration and handle HTTPS

### PR DESCRIPTION
**Pull request by rodrigogansobarbieri**
Closes: #202

### CODE REVIEW

File: openstack/tools/enable_samltestid.sh [hunk 1]
```diff
@@ -5,10 +5,21 @@
 credentials=$(env | grep OS_ || true)
 
 if [[ -z $credentials ]];then
-    echo "Missing cloud credentials"
+    echo "Missing overcloud credentials"
     exit 1
 fi
 
+if [[ $OS_AUTH_URL == *keystone.ps6* ]] || [[ $OS_AUTH_URL == *10.230* ]];then
```
Comment (nicolasbock) [inline openstack/tools/enable_samltestid.sh:12]: You could also use


```suggestion
if [[ $OS_AUTH_URL =~ keystone.ps6 ]] || [[ $OS_AUTH_URL =~ 10.230 ]];then
```

File: openstack/tools/enable_samltestid.sh [hunk 2]
```diff
@@ -5,10 +5,21 @@
...
+    echo "Use the overcloud credentials, not undercloud"
+    exit 1
+fi
+
+HTTPS=false
+
+if [[ $OS_AUTH_URL == https://* ]];then
```
Comment (nicolasbock) [inline openstack/tools/enable_samltestid.sh:19]: Same here


```suggestion
if [[ $OS_AUTH_URL =~ ^https:// ]];then
```

**Review by pponnuvel — APPROVED** [review]
LGTM

File: openstack/tools/enable_samltestid.sh (hunk 1 above)
Comment (rodrigogansobarbieri) [reply openstack/tools/enable_samltestid.sh]: Done

File: openstack/tools/enable_samltestid.sh (hunk 2 above)
Comment (rodrigogansobarbieri) [reply openstack/tools/enable_samltestid.sh]: Done

**Review by nicolasbock — APPROVED** [review]
Thanks @rodrigogansobarbieri . LGTM

**Pull request merged**